packages/config-cli/src/cli/screenshot/screenshot.ts

```diff
@@ -0,0 +1,143 @@
+import { Config, fsa, LogConfig, LogType } from '@basemaps/shared';
+import { mkdir } from 'fs/promises';
+import { Browser, chromium } from 'playwright';
+import { CommandLineAction, CommandLineFlagParameter, CommandLineStringParameter } from '@rushstack/ts-command-line';
+
+interface Location {
+  lat: number;
+  lng: number;
+  z: number;
+}
+interface TileTest {
+  name: string;
+  tileMatrix: string;
+  location: Location;
+  tileSet: string;
+  style?: string;
+}
+
+export class CommandScreenShot extends CommandLineAction {
+  private host: CommandLineStringParameter;
+  private tag: CommandLineStringParameter;
+  private tiles: CommandLineStringParameter;
+  private verbose?: CommandLineFlagParameter;
+
+  public constructor() {
+    super({
+      actionName: 'screenshot',
+      summary: 'dump screenshots of from LINZ Basemaps',
+      documentation: 'Dump screenshots with selected tile sets',
+    });
+  }
+
+  protected onDefineParameters(): void {
+    this.host = this.defineStringParameter({
+      argumentName: 'HOST',
+      parameterLongName: '--host',
+      description: 'Host to use',
+      defaultValue: 'basemaps.linz.govt.nz',
+    });
+
+    this.tag = this.defineStringParameter({
+      argumentName: 'TAG',
+      parameterShortName: '-t',
+      parameterLongName: '--tag',
+      description: 'PR tag(PR-number) or "production"',
+      defaultValue: 'production',
+    });
+
+    this.tiles = this.defineStringParameter({
+      argumentName: 'TILES',
+      parameterLongName: '--tiles',
+      description: 'JSON file path for the test tiles',
+      defaultValue: './test-tiles/default.test.tiles.json',
+    });
+
+    this.verbose = this.defineFlagParameter({
```

Comment:
verbose is handled by being a part of `BaseCommandLine` 

`./bmc --verbose screenshot`

Reply:
All fixed. Can you give another look?